packages/indy-vdr/src/dids/IndyVdrIndyDidRegistrar.ts

```diff
@@ -76,26 +78,62 @@ export class IndyVdrIndyDidRegistrar implements DidRegistrar {
             },
           }
         }
-        if (!isSelfCertifiedIndyDid(did, verkey)) {
-          throw new Error(`Initial verkey ${verkey} does not match did ˇ${did}`)
+
+        const { namespace, id } = parseIndyDid(did)
```

Comment:
I see it's used afterwards, but maybe would be nice to use directly here the same naming than Anoncreds Registry class, as it is more explicit.

Reply:
I though I understood this comment, but I don't understand. Do you mean to also use `namespaceIdentifier` here?

Reply:
Yes, basically I was referring to that. But just a bit nitpicking there, I think it's still clear.